fedocal/__init__.py

```diff
@@ -864,9 +870,30 @@ def add_meeting(calendar_name):
             day=form.meeting_date.data.day))
     elif flask.request.method == 'GET':
         form = forms.AddMeetingForm(timezone=tzone, calendars=calendars)
+        mtg_date = flask.request.args.get('date', None)
+        if mtg_date:
+            form.meeting_date.data = parser.parse(mtg_date).date()
+            form.meeting_date_end.data = parser.parse(mtg_date).date()
+
+        mtg_time = flask.request.args.get('time', None)
+        if mtg_time:
+            form.meeting_time_start.data = mtg_time
+            hours, minutes = mtg_time.split(':')[:2]
+            try:
+                hours = int(hours) + 1
```

Comment:
For the template logic: https://github.com/fedora-infra/fedocal/pull/168/files#diff-38faee44bdff0a8e66fb3ddb82715945R91